processor/opentelemetry-processor-baggage/tests/test_baggage_processor.py

```diff
@@ -59,7 +127,9 @@ def test_set_baggage_attaches_to_child_spans_and_detaches_properly_with_token(
         self,
     ):
         tracer_provider = TracerProvider()
-        tracer_provider.add_span_processor(BaggageSpanProcessor())
+        tracer_provider.add_span_processor(
+            BaggageSpanProcessor(ALLOW_ALL_BAGGAGE_KEYS)
```

Comment:
If this is a required argument, then it should be logged as a breaking change

Reply:
Yeah, it is a breaking change in that we've introduced a required parameter. However, this processor hasn't been released yet so was trying to lean on not declaring a breaking change 😄

Reply:
I guess so long as it's merged before the release, this should be ok